src/routes/reference/melbands.svx

```diff
@@ -1,8 +1,86 @@
 ---
 title: MelBands
-blurb: Amplitude for a number of equally spread perceptual bands
+blurb: Magnitudes for a number of perceptually-evenly spaced spectral bands
```

Comment:
Personally I'd hedge 'perceptually-evenly spaced' because the mel scale is an approximation, and possibly not even an especially good one. Expectation management n' that